doc/man1/openssl-s_server.pod.in

```diff
@@ -762,11 +763,17 @@ Protocol names are printable ASCII strings, for example "http/1.1" or
 "spdy/3".
 The flag B<-nextprotoneg> cannot be specified if B<-tls1_3> is used.
 
+=item B<-ktls>
+
+Enable Kernel TLS for sending and receiving.
+This option was introduced in OpenSSL 3.1.0.
+Kernel TLS is off by default as of OpenSSL 3.1.0.
+
 =item B<-sendfile>
 
 If this option is set and KTLS is enabled, SSL_sendfile() will be used
 instead of BIO_write() to send the HTTP response requested by a client.
-This option is only valid if B<-WWW> or B<-HTTP> is specified.
+This option is only valid if B<-ktls> and B<-WWW> or B<-HTTP> is specified.
```

Comment:
I'd probably remove the `If....` above.

Reply:
And it should be something like:

`This option is only valid if B<-ktls>, B<-WWW> or B<-HTTP> is specified.`

The `and` should be removed as well.

Reply:
> And it should be something like:
> 
> `This option is only valid if B<-ktls>, B<-WWW> or B<-HTTP> is specified.`
> 
> The `and` should be removed as well.

No, the condition really is that -ktls must be used and one of the -WWW or -HTTP must be used.

Reply:
Ahh, that would be a little confusing. Should be better to say:

`...if B<-ktls> along with B<-WWW> or B<-HTTP> is specified.`